pkg/services/scm_provider/gitlab.go

```diff
@@ -0,0 +1,135 @@
+package scm_provider
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+type GitlabProvider struct {
+	client       *gitlab.Client
+	organization string
```

Comment:
Should probably rename this to `group` for consistency.

```suggestion
	group string
```